charts/seed-bootstrap/aggregate-prometheus-rules/fluent-bit.rules.yaml

```
@@ -38,6 +38,18 @@ groups:
         description: >
           The input plugins of Fluent-bit pod {{$labels.pod}} running on seed {{$externalLabels.seed}} haven't collected any logs for the last 6 hours.
 
+    - alert: NodeKubeletDown
+      expr: ALERTS{alertname="NodeKubeletDown", alertstate="firing"} > 0
+      for: 10m
+      labels:
+        service: node-exporter
+        severity: warning
+        type: seed
+        visibility: owner
+      annotations:
+        summary: Nodes not ready for more than 10 minutes 
+        description: Shows the number of node that are in the NotReady state for more than 10 minutes du to the kubelet not responding
```

```suggestion
        description: Shows the number of nodes that are in the NotReady state for more than 10 minutes due to the kubelet not responding
```

Please watch out for typos! :)

- What is the effect of `> 0` in the expression above?
- If this alert is supposed to be a summary of all the shoots (in the seed, later even on a higher level), then an aggregation is missing - in its current form, it emits the same number of alerts as its input.

---

Responding in part:
- Typos fixed !
- `> 0` no longer relevant - the correct expression is posted in a comment and fixed in the code.